### PR TITLE
Create a file for Södertälje Eateries

### LINKED ISSUE #6: silverfisk's recommendations for things to do/eat
## Overview
- [ ]  Create branch
- [ ]  Add my file
- [ ]  Commit my file
- [ ]  Push to GitHub
- [ ]  Open a pull request
- [ ]  Collaborate
- [ ]  Merge

### PR DESCRIPTION
**Pull request by silverfisk**
This PR closes #6 by adding a Sodertalja file.

### CODE REVIEW

**Review by johsto — COMMENTED** [review]
Any good pizza places to add?

File: silverfisk-sodertalje.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Södertälje / Stockholm
+### Great places to eat
```
Comment (githubteacher) [inline silverfisk-sodertalje.md:2]: The hotel recommended an Italian place in City Center - have you been there?

File: silverfisk-sodertalje.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+### Fun things to do
```
Comment (githubteacher) [inline silverfisk-sodertalje.md:6]: Especially things that are still open after 4 PM?

**Review by githubteacher — CHANGES_REQUESTED** [review]

File: silverfisk-sodertalje.md (hunk 1 above)
Comment (silverfisk) [reply silverfisk-sodertalje.md:2]: No, dont think so. Do you have the name of the place?

**Review by silverfisk — COMMENTED** [review]
Seems reasonable

**Pull request merged**